src/test/set/Set.t.sol

```diff
@@ -62,30 +81,18 @@ contract SetTest is DSTest {
         uint256 rollupBalanceDpi = dpi.balanceOf(address(rollupProcessor));
         uint256 rollupBalanceEth = address(rollupProcessor).balance;
 
-        assertEq(
-            depositAmountDai,
-            rollupBalanceDai,
-            "DAI balance must match"
-        );
-        
-        assertEq(
-            depositAmountDpi,
-            rollupBalanceDpi,
-            "DPI balance must match"
-        );
-        
-        assertEq(
-            depositAmountEth,
-            rollupBalanceEth,
-            "ETH balance must match"
-        );
+        assertEq(depositAmountDai, rollupBalanceDai, 'DAI balance must match');
+
+        assertEq(depositAmountDpi, rollupBalanceDpi, 'DPI balance must match');
+
+        assertEq(depositAmountEth, rollupBalanceEth, 'ETH balance must match');
     }
 
     function testIssueSetForExactToken() public {
         // Pre-fund contract with DAI
         uint256 depositAmountDai = 1000000000000000000000; // $1000
```

Comment:
Notationwise it would be easier to read `1000 ether` or `1000e18` for the value.